week-4/A-objects-intro/README.md

```diff
@@ -0,0 +1,9 @@
+### Goals
+
+- introduce objects as a concept
+- when can we use objects (to store what type of information)
+- the very specific object syntax (the commas after each property, the colon between property name and value, the curly brackets)
+- the language to use when talking about an objects: object literal, properties, property names, property values
+- reading object properties using the *dot notation*: `console.log(myObject.name)`
+- setting object properties: `myObject.name = "John"`
+- very simple exercises to get them used to declaring object literals, reading and writing from/to them

```

Comment:
Also rules for naming property names (not having spaces, what to do when there is space ...)